test/experiments/scheduler_mocking/experiments_mocks.py

```diff
@@ -89,21 +89,21 @@ def get_expected_experiment_models(
         "expected": [
             ExperimentModel(
                 _id=None,
-                experiment_id=20310000,
+                experiment_id=str(20310000),
```

Comment:
Instead of wrapping the integer in `str()`, can you add double quotes around each number to turn them into strings in the code please?

Reply:
changed